include/ghex/unstructured/util/test_util.hpp

```diff
@@ -0,0 +1,439 @@
+﻿/*
```

Comment:
This file contains instances of grids, right? this could be better placed in `tests` folder?

Reply:
I'd better move it there, yes. Also, it makes use of the GTEST facilities. Should I use a dedicated namespace for it?

Reply:
maybe, up to you. As the file is only used by one test file the namespace is not really necessary, I would say

Reply:
Ok, done.